src/frontend/context/DokumentutsendingContext.ts

```diff
@@ -203,11 +206,11 @@ export const [DokumentutsendingProvider, useDokumentutsending] = createUseContex
             };
         };
 
-        const hentBarnSøktForSkjemaData = (
+        const hentRelevanteBarnSkjemaData = (
             brevmal: Informasjonsbrev,
             målform: Målform
         ): IManueltBrevRequestPåFagsak => {
-            const barnIBrev = skjema.felter.barnSøktFor.verdi.filter(barn => barn.merket);
+            const barnIBrev = skjema.felter.relevanteBarn.verdi.filter(barn => barn.merket);
```

Comment:
Hadde det vært en idé å kalle det `barnIBrev` i stedet for `relevanteBarn` over alt? Har ingen sterke preferanser altså, men syns det ga litt mening når jeg så det her

Reply:
Ja, barnIBrev tror jeg funker fint 😊